pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -12,19 +12,66 @@ To learn more about defining configuration parameters, see xref:policies-pdk-cre
 
 == Before You Begin
 
-* xref:policies-pdk-publish-policies.adoc[Upload your Custom Policy to Exchange]
+* xref:policies-pdk-compile-policies.adoc[Compile your custom policy].
+* If applying you policy to Flex Gateway running in Local Mode, xref:policies-pdk-publish-policies.adoc[upload your custom policy to exchange].
```

Comment:
I think there's a typo here: you => your

Also, were you referring here to Connected Mode?